shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -302,18 +303,30 @@ class DiffTests extends org.scalatest.funsuite.AnyFunSuite with org.scalatest.Pa
             typer.dbg = mode.debugVariance
             typer.computeVariances(curBlockTypeDefs, ctx)
             typer.dbg = temp
-            
+            val varianceWarnings: MutSortedMap[Str, Ls[Str]] = MutSortedMap()
```

Comment:
Instead of a `Ls[Str]`, better keep `TypeName`s here, which also carry some location info.